tests/home-page-check

```diff
@@ -0,0 +1,9 @@
+import { test } from "@framework/test";
+
+test.describe("Verify GuardianUI.com home page", () => {
+	test("Should verify that the target page loads", async({ page, dappPage }) => {
```

Comment:
@lipmaneth see comment about dropping `dappPage` in order to avoid pulling the proprietary test framework.